policy/modules/system/systemd.te

```diff
@@ -1377,6 +1387,10 @@ init_runtime_filetrans(systemd_resolved_t, systemd_resolved_runtime_t, dir)
 
 dev_read_sysfs(systemd_resolved_t)
 
+fs_getattr_cgroup(systemd_resolved_t)
```

Comment:
Please double check the fs getattr rules, as some of the domains have `fs_getattr_all()` but unfortunately the lines are out of order.

Reply:
Good call. There were some duplicate rules added since the `fs_getattr_all()`s were added some time after I originally worked on this.